qiskit_experiments/framework/backend_data.py

```diff
@@ -0,0 +1,145 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Backend data access helper class
+
+Since `BackendV1` and `BackendV2` do not share the same interface, this
+class unifies data access for various data fields.
+"""
+from qiskit.providers import BackendV1, BackendV2
+
+
+class BackendData:
+    """Class for providing joint interface for accessing backend data"""
+
+    @staticmethod
+    def name(backend):
+        """Returns the backend name"""
+        if isinstance(backend, BackendV1):
+            return backend.name()
+        elif isinstance(backend, BackendV2):
+            return backend.name
+        return ""
+
+    @staticmethod
+    def control_channel(backend, qubits):
```

Comment:
I'm fine with removing this if the method is only used by CR Ham tomo experiment. Basically `control_channels` dict gives you full set of control channel mapping so the tomo experiment should be able to get the control channel object with that dictionary.

Reply:
This one is here because `BackendV2` has the exact same method, and I want it to be as similar as possible. I'm ok with keeping it.